src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form action="text" class="big-search"
+     data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      action="#"
      class="big-search"
      data-qa="big"
    >
```